packages/block-library/src/gallery/edit.js

```diff
@@ -52,7 +52,7 @@ const PLACEHOLDER_TEXT = Platform.select( {
 	web: __(
 		'Drag images, upload new ones or select files from your library.'
 	),
-	native: __( 'ADD MEDIA' ),
+	native: __( 'Add Media' ),
```

Comment:
This one would probably still be good to fix regardless. I'm not really sure why this label is fully-capitalized?

cc @mkevins @pinarol @jorgefilipecosta re: #18265

If it's a UI thing, I'd wonder if there's some native equivalent to the web's [`text-transform: uppercase`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform).

Reply:
This all-caps string was brought in by this commit: https://github.com/WordPress/gutenberg/pull/18265/commits/abd295d3ac1d358fef055664c93d00baf34a8d72 to be consistent with the other native media blocks: https://github.com/WordPress/gutenberg/blob/78de49ede8fac2cf76600e9048d653d56ab30c11/packages/block-editor/src/components/media-placeholder/index.native.js#L75-L83

I believe the `text-transform` property (`textTransform` on React Native) is [now supported](https://facebook.github.io/react-native/docs/text#style) on both iOS and Android, so the native `MediaPlaceholder` could be refactored to use that. I'm not :100: sure, but it may be that this was previously not possible (at the time these strings were introduced).

Reply:
> I'm not really sure why this label is fully-capitalized?

It was a design decision. 

> I believe the text-transform property (textTransform on React Native) is now supported on both iOS and Android, so the native MediaPlaceholder could be refactored to use that. I'm not 💯 sure, but it may be that this was previously not possible (at the time these strings were introduced).

This is tricky mostly because of non technical reasons. Our translation rates aren't perfect in some languages. Thus, if we use device locale to do the uppercasing, it might generate some unexpected results on strings that are not localized yet. In general, we don't usually do uppercasing/lowercasing for user facing Strings on mobile.

<img width="200" alt="portfolio_view" src="https://user-images.githubusercontent.com/5032900/74633355-805e3280-5172-11ea-99fa-e1cde7e159b6.jpg">

Reply:
> This is tricky mostly because of non technical reasons. Our translation rates aren't perfect in some languages. Thus, if we use device locale to do the uppercasing, it might generate some unexpected results on strings that are not localized yet.

Thanks for that extra context!

Reply:
@pinarol I'm a bit unclear on whether you're suggesting we cannot or should not use `textTransform`? In my mind, the device transform would have the same result as the current implementation whether or not the string is translated, and I would think it would be preferable to err on the side of not ascribing that case in the string used in code (for consistency, and for improved reusability of that string).

Reply:
In some languages the uppercase of some letter is different than their uppercase in English. Like in Turkish, the uppercase of "i" is "İ", and the lowercase of "I" is "ı". So if we programmatically uppercase "Add Media" using TR locale it'll become "ADD MEDİA" which is not correct. This wouldn't be a problem if we had good translation support on every language but in some languages users see a mixture of English & X language words. So we'll be uppercasing English words using X locale, which is why I want to avoid using dynamic uppercasing/lowercasing.

But I agree that we should aim for improved reusability, so let me loop in @iamthomasbishop and see what he thinks about dropping the fully uppercase "ADD MEDIA" in favor of "Add Media"?

Reply:
I would definitely prefer writing the strings in one way (sentence-case to align w/ Core) and programatically transform based on the target platform's component/label guidelines.

In other words, we'd always write the string as sentence-case (for example: `Xxx xxxxx` or `Add media`). From there, we would transform casing on RN based on the platform guidelines for the corresponding component (button == uppercase on Android and title-case on iOS, whereas a normal text-label title would be sentence-case on Android and title-case on iOS).

Does this make sense? I also remember someone mentioning that we aren't able to reliably use `text-transform` on RN, but not sure if that's resolved.